components/brave_wallet/browser/asset_ratio_service.cc

```diff
@@ -265,4 +265,45 @@ void AssetRatioService::OnGetTokenInfo(
       ParseTokenInfo(body, mojom::kMainnetChainId, mojom::CoinType::ETH));
 }
 
+// static
+GURL AssetRatioService::GetCoinMarketsURL(const std::string& vs_asset,
+                                          const uint8_t limit) {
+  std::string spec = base::StringPrintf(
```

Comment:
seems this should be using `AppendQueryParameter`, per https://github.com/brave/brave-browser/issues/24401